autrainer/training/training.py

```diff
@@ -211,33 +220,18 @@ def __init__(
         self.max_dev_metric = self.data.tracking_metric.starting_metric
         self.best_iteration = 1
 
-        # ? Save initial (and best) Model, Optimizer and Encoder State
-        self.bookkeeping.create_folder("_initial")
-        self.bookkeeping.save_state(self.model, "model.pt", "_initial")
-        self.bookkeeping.save_state(self.optimizer, "optimizer.pt", "_initial")
-        if self.scheduler:
-            self.bookkeeping.save_state(
-                self.scheduler, "scheduler.pt", "_initial"
-            )
-
-        self.bookkeeping.create_folder("_best")
-        self.bookkeeping.save_state(self.model, "model.pt", "_best")
-        self.bookkeeping.save_state(self.optimizer, "optimizer.pt", "_best")
-        if self.scheduler:
-            self.bookkeeping.save_state(
-                self.scheduler, "scheduler.pt", "_best"
-            )
-
-        self.bookkeeping.save_audobject(
-            self.data.target_transform, "target_transform.yaml"
-        )
-        self.bookkeeping.save_audobject(self.model, "model.yaml")
-        self.bookkeeping.save_audobject(
-            self.data.test_transform, "inference_transform.yaml"
-        )
-        self.bookkeeping.save_audobject(
-            self.data.file_handler, "file_handler.yaml"
-        )
+        # ? Save initial (and best) Model, Optimizer and Scheduler states
+        [
```

Comment:
Won't it be easier to understand if you turn the list comprehension in a simple for loop?